Source/WebCore/page/scrolling/ScrollAnchoringController.cpp

```diff
@@ -25,30 +25,68 @@
 
 #include "config.h"
 #include "ScrollAnchoringController.h"
+#include "ElementIterator.h"
+#include "HTMLHtmlElement.h"
+#include "RenderBox.h"
+#include <wtf/text/TextStream.h>
 
 namespace WebCore {
 
 void ScrollAnchoringController::invalidateAnchorElement()
 {
     m_anchorElement = nullptr;
-    m_lastPositionForAnchorElement = FloatPoint();
+    m_lastOffsetForAnchorElement = { };
 }
 
-void ScrollAnchoringController::selectAnchorElement()
+FloatSize ScrollAnchoringController::computeOffset(RenderBox* candidate)
```

Comment:
Nit - perhaps this helper should take a `const RenderBox&` since it expects a non-null `RenderBox` (and doesn't mutate it in any way)

Reply:
Fixed.